### DISCUSSION TITLE
有个mysql大表查询适合换成Doris吗

### DISCUSSION THREAD

**Discussion by luger1990** [apache/doris#25000] [RFC]
我们有个数据库最大的表大概有5KW条数据，原始数据是用mysql存储的，然后试试导入到es中为管理平台提供查询，我在想是否适合把数据同步到Doris给管理平台做查询。我们用的都是比较简单的查询。

**Comment by LemonLiTree** [comment]
completely fine

**Reply by echobigdata** [reply]
非常合适
